### ISSUE TITLE
Auto 10 minute release time on every upload. This way all file transfers and syncing gets a chance before someone plays it.

### ISSUE THREAD

**Issue by JoshWho** [WWBN/AVideo#9637]
I noticed out of a lot of tests today, if you try to view a newly uploaded video before it gets finished transferring to the cdn, the video breaks. 

Sometimes notifications tell us the upload is ready and then go to watch it within a few seconds from getting the email and the video breaks and gets marked broken.

I think the only way to prevent this is to give a delay on the release time like 10 minutes or so.

**Comment by JoshWho** [comment]
> It breaks, but the video is not lost
> 
> If you wait it to finish you will see a good video.
> 
> Also I guess the issue is something else, maybe the video still unzipping the HLS files.

Yeah, just will be best to give it a few more minutes before emailing everyone that way it will be playable when people do get the email.

**Comment by eitanwa** [comment]
I think it's a great idea
The emails are sent at a very early stage
Even in articles, just opening an article sends an email, even before you've written a sentence
And even if in the end you decided not to publish at all

(Basically there is a solution for this to pre-set the video/article as unavailable) and only after finishing change it to available for viewing

The solution that Gosh suggests to pause a little is better in my opinion

In addition, there is another matter I wrote about in the past, I thought that maybe in the last updates it would change

In every video editing,
In each of its parameters: name, description, category, etc
A message is sent that a new video has been uploaded
This also happens when editing a video that was uploaded several years ago (not only in new ones)
I don't know if it's supposed to be that way, I don't think so
And maybe it only happens to me, so I'd love the comments of others maybe @josh 
tnks

**Comment by JoshWho** [comment]
Now starting to get double emails again and I think this delay will also solve that issue as well.

**Comment by DanielnetoDotCom** [comment]
I disagree, I think the emails must be sent in the right time, when the video is available. 

we need to detect why it is generating a broken missing file or why it is not playable.

**Comment by JoshWho** [comment]
U said it is not really broken. and after a few minutes it works again. I think when it starts working should be when to send it out.

**Comment by DanielnetoDotCom** [comment]
yes, so we need to find why is it saying the "fake" broken

I do not think is the sync, it will only sync when all the files are in the storage, before that no files will be deleted. 

If you find a way to reproduce it or find a exact time when it become broken I may be able to find something in the logs.

**Comment by JoshWho** [comment]
Just upload a 2 hour video wait for it to say it is uploaded and click play

**Comment by JoshWho** [comment]
Rumble actually does this they make a delay on the email just incase files are not done encoding.

**Comment by eitanwa** [comment]
Maybe in the same matter of privacy
When a channel sends an email to its followers
The received indication is that an email was sent from the site
 support in my case (the email in the website settings)
And there is no indication that the email was sent from the specific channel

**Comment by JoshWho** [comment]
> Maybe in the same matter of privacy When a channel sends an email to its followers The received indication is that an email was sent from the site support in my case (the email in the website settings) And there is no indication that the email was sent from the specific channel

Yes I think he just fixed that template last week. Should now say the channel name that sends it.

**Comment by eitanwa** [comment]
> > Maybe in the same matter of privacy When a channel sends an email to its followers The received indication is that an email was sent from the site support in my case (the email in the website settings) And there is no indication that the email was sent from the specific channel
> 
> Yes I think he just fixed that template last week. Should now say the channel name that sends it.

I tried just before I sent the message

**Comment by eitanwa** [comment]
> ahhhh, Did u do a git pull recently?

today

**Comment by JoshWho** [comment]
![🔴JoshWho-Truth-Intel-TV-Learn-what-the-Gov-doesn-t-want-you-to-know-SeekingTheTruth-Live-Stream-·-FTJ-Media-12-11-2024_01_48_AM](https://github.com/user-attachments/assets/3f97155f-b357-4fbe-a682-ac932bd3c7a3)

**Comment by JoshWho** [comment]
This delay is a must for real @DanielnetoDotCom

**Comment by JoshWho** [comment]
We are now blocked from Outlook domains because of this. THIs has to be fixed.

**Comment by JoshWho** [comment]
What is so wrong about delaying it until ready? This is blowing my mind.

**Comment by eitanwa** [comment]
@JoshWho 
Please note
Beyond the issue of sending the amount of emails and their speed

There is an important issue that you should know
I noticed this recently
I wrote it earlier in this post
But surely it passed you by because of the previous issue which is very important

However, as an admin, every edit you make to a video
(I didn't go through all the parameters because there are a lot)
But what I can say for sure
That every edit of a video name, video description, status, photo, owner, etc. ...
Sends an email to the subscribers!!!!

Now I'm also checking this as a user who is not an admin
(Do the editing changes he makes also send an email)
In the meantime, from what I've been able to test and can say it's different

I believe it's important to know this
At least it will reduce the amount of emails sent
(Beyond those in the main subject of the post)

**Comment by eitanwa** [comment]
> Must reinforce what Josh says about the amount of three miles Beyond that There is a privacy issue As soon as you upload a video or article even before you finish editing or even saving, an email is sent
> 
> Example: I upload a video or an article that I don't want to be visible And I'm still in the editing stage (even before saving) At this time the email has already been sent And anyone can enter it (even though I set it to not be visible)



Returning from my words
 the emails work as expected
That's was the problem 
It should be unlisted on default!!!  At least not on active mode!!
@JoshWho 

![image](https://github.com/user-attachments/assets/039457ee-2357-4c28-b950-2c34dd05fc56)

The sending speed is excellent!!
For the reason that only the admin receives the e-mail that has "not been written yet"
And that way he can be monitored for what happens on the site
In the initial state that it is not registered, an email will not be sent to followers except for the admin (website administrator)
And it's important, it's exactly the opposite of what I wrote before ( about the speed)

question:
In these definitions there is also unpublished
What is the meaning?
I also tried this setting before the unlisted setting
And it was published and emails were sent

Thanks Daniel

p.s
In **most** cases, the site administrator will receive at least 2 emails
not the users

**Comment by eitanwa** [comment]
And another important thing, the "delay" is excellent in this situation

**Comment by eitanwa** [comment]
Please do not refer to the previous 2 messages I sent
I tried to fix and edit
and by mistake this is what came out (can't delete and edit)
I will update

**Comment by JoshWho** [comment]
I can confirm the double sometimes quadruple emails only happens when the cpu is maxed out.

**Comment by DanielnetoDotCom** [comment]
Hi
I still do not think the 10 minute release is a good solution
I got something regarding the emails. 
I am sending updates to the Notification plugins now

@JoshWho your site is up to date. please keep the eyes opened.

**Comment by JoshWho** [comment]
Thank you @DanielnetoDotCom
